pkg/component/component.go

```diff
@@ -202,28 +202,6 @@ func GetComponentTypeFromDevfileMetadata(metadata devfile.DevfileMetadata) strin
 	return componentType
 }
 
-// GetProjectTypeFromDevfileMetadata returns component type from the devfile metadata
-func GetProjectTypeFromDevfileMetadata(metadata devfile.DevfileMetadata) string {
-	var projectType string
-	if metadata.ProjectType != "" {
-		projectType = metadata.ProjectType
-	} else {
-		projectType = NotAvailable
-	}
-	return projectType
-}
-
-// GetLanguageFromDevfileMetadata returns component type from the devfile metadata
-func GetLanguageFromDevfileMetadata(metadata devfile.DevfileMetadata) string {
-	var language string
-	if metadata.Language != "" {
-		language = metadata.Language
-	} else {
-		language = NotAvailable
-	}
-	return language
-}
```

Comment:
IIRC, these functions were used for `odo push`'s telemetry part. We are going to need this again because we will need telemetry for `odo dev` as well. ATM, `odo dev` doesn't have telemetry (which occurred to me when I saw these functions being removed.)

I don't know if we should keep these functions for above stated future use or not, I said this just as a note.

Reply:
Right, it seems safer to keep them